sw/device/sca/aes_serial.c

```diff
@@ -167,13 +169,17 @@ static void aes_send_ciphertext() {
   dif_aes_data_t ciphertext;
   SS_CHECK_DIF_OK(dif_aes_read_output(&aes, &ciphertext));
 
-  simple_serial_send_packet('r', (uint8_t *)ciphertext.data, kAesTextLength);
+  if (only_first_word) {
+    simple_serial_send_packet('r', (uint8_t *)ciphertext.data, 4);
+  } else {
+    simple_serial_send_packet('r', (uint8_t *)ciphertext.data, kAesTextLength);
+  }
```

Comment:
Thanks for making this configurable. It's good to have the option to still transmit more in case we should run into issues in the future.